ngrok.js

```diff
@@ -1,67 +1,88 @@
-const ng = require('ngrok');
+const ng = require('@ngrok/ngrok');
+const Package = require('./package.json');
 
 module.exports = function (RED) {
   function ngrok(config) {
-    const region_types = ["us", "eu", "ap", "au", "sa", "jp", "in"];
-    const proto_types = ["http", "tcp"];
-    const bind_tls_types = ["https", "true", "http", "false", "both"];
+    const region_types = ['us', 'eu', 'ap', 'au', 'sa', 'jp', 'in'];
```

Comment:
PM from ngrok here. region is now optional with the release of our global routing. you can still use region if you'd like, but we have also added a new one, `us-cal-1` https://ngrok.com/docs/network-edge/#points-of-presence

Reply:
thx, add ```us-cal-1```

Reply:
@russorat What effect does the region now have? Just wondering if there's any point in allowing the user to configure it now?

Reply:
good question. users can still force the agent to connect to specific region if they like using this flag. Paid users can also create endpoints that route to a specific region instead of the closest region if they like.

Reply:
@russorat From what I can see though this flag has no effect on either:
The server which the client connects to 
Where the URL for the ngrok hostname resolves to.

In my tests setting the region to either Frankfurt, California or Sydney the client still established a connection to an AWS server in Frankfurt (that would be my closest as I'm in the UK)
And the hostname that was allocated `57d9da1e1be2.ngrok.app` always resolved via GeoIP to multiple POPs depending on where the query was coming from.

In the previous client setting the region flag gave a geographic hostname eg `sammachin.eu.ngrok.io` which meant that HTTP connections were terminated in that region,

With this new client there seems to be no point in setting the region flag.

Reply:
@sammachin sorry about this. It looks like the region flag is not supported in the ngrok-javascript lib (i misspoke because it is still supported in the ngrok-go lib but only because it was written earlier).

We still have the ability to connect to a specific region if you like but only through explicitly setting the server address you are connecting through. See the example here: https://ngrok.com/docs/network-edge/gslb/?cty=javascript-sdk#overview

So in this case, you can still support region types, but it would set `.serverAddr("connect.<REGION>.ngrok-agent.com:443")`.

Again, sorry for the miscommunication.